pom.xml

```diff
@@ -426,7 +426,7 @@
     <testcontainers.version>1.13.0</testcontainers.version>
     <jedis.version>3.2.0</jedis.version>
     <confluent.version>5.5.0</confluent.version>
-    <confluent-ce.version>5.5.1-ce</confluent-ce.version>
+    <confluent-ce.version>5.5.0-ce</confluent-ce.version>
```

Comment:
why is this downgrade required?

Reply:
I don't think it is. But I think confluent and confluent-ce versions should be identical